pcgl2cache/app/v1/routes.py

```diff
@@ -15,6 +16,40 @@
 )
 
 
+def remap_public(func=None, *, check_node_ids=False):
+    def mydecorator(f):
+        @wraps(f)
+        def decorated_function(*args, **kwargs):
+            virtual_tables = current_app.config.get("VIRTUAL_TABLES", None)
+
+            # if not virtual configuration just return
+            if virtual_tables is None:
+                return f(*args, **kwargs)
+            table_id = kwargs.get("table_id", None)
+
+            if table_id is None:
+                # then no table remapping necessary
+                return f(*args, **kwargs)
+            if not table_id in virtual_tables:
+                # if table table_id isn't in virtual
+                # tables then just return
+                return f(*args, **kwargs)
+            else:
```

Comment:
think the `else` is not necessary, otherwise lgtm

Reply:
fixed

Reply:
@akhileshh can you give it a review so it can be merged?